src/script/view_model/CallingViewModel.ts

```diff
@@ -81,7 +83,7 @@ declare global {
     setSinkId?: (sinkId: string) => Promise<void>;
   }
 }
-const maxGroupSize = 4;
+const MAX_USERS_TO_CALL_WITHOUT_CONFIRM = 1;
```

Comment:
```suggestion
const MAX_USERS_TO_CALL_WITHOUT_CONFIRM = 4;
```

Not against having the modal show up every time, but 1 is probably a little low :D

Reply:
https://github.com/wireapp/wire-webapp/pull/16248/commits/c45196d1e882235a49cc69987d8cf9ed2b5f422f